libs/gltfio/src/MaterialGenerator.cpp

```diff
@@ -126,10 +129,43 @@ static std::string shaderFromKey(const MaterialKey& config) {
                 shader += "metallicRoughnessUV = (vec3(metallicRoughnessUV, 1.0) * "
                         "materialParams.metallicRoughnessUvMatrix).xy;\n";
             }
+            if (config.useSpecularGlossiness) {
+                shader += R"SHADER(
+                    vec4 sg = texture(materialParams_metallicRoughnessMap, metallicRoughnessUV);
+                    glossiness *= sg.a;
+                    vec3 specColor = sg.rgb * materialParams.specularFactor;
+
+                    // NOTE: this conversion logic is a carry-over from MeshAssimp. According to the
+                    // spec: "The specular property from specular-glossiness material model is the
+                    // same as the base color value from the metallic-roughness material model for
+                    // metals."
+                    material.reflectance = 0.5;
```

Comment:
We shouldn't set reflectance. 0.5 is the default.

Reply:
Will do.